jenkins/lava-boot-v2.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env python3
+# -*- coding:utf-8 -*
+#
+
+import os
+from subprocess import Popen
+
+labs = {
+    'lab-free-electrons': {
+        'blacklist': [
+            'android'
+        ],
+    },
+}
+
+# Basic env
+LAB = os.environ['LAB']
```

Comment:
Might be worth a comment to say this is only there to reproduce the environment produced by Jenkins.